packages/manager/modules/vps/src/dashboard/vps-dashboard.html

```diff
@@ -126,15 +126,17 @@
                 </div>
                 <div
                     class="oui-tile__item oui-tile__item_button"
-                    data-ng-if="$ctrl.hasFeature($ctrl.DASHBOARD_FEATURES.upgrade) && $ctrl.actions.upgrade.state && $ctrl.actions.upgrade.isAvailable() && !$ctrl.isMigrating"
+                    data-ng-if="$ctrl.canBeMigrated()"
```

Comment:
already exists and we forgot to use it ?

Reply:
Yes Marie it was existing one I just used the same to show the migrate option.